<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMasterPasswordResetDialog</class>
 <widget class="QDialog" name="QgsMasterPasswordResetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reset Master Password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter CURRENT master authentication password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsPasswordLineEdit" name="leMasterPassCurrent">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Required</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter NEW master authentication password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsPasswordLineEdit" name="leMasterPassNew">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Required</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Re-enter NEW password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsPasswordLineEdit" name="leMasterPassNew2">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Required</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkKeepBackup">
     <property name="text">
      <string>Keep backup of current database</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblWarning">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leMasterPassCurrent</tabstop>
  <tabstop>leMasterPassNew</tabstop>
  <tabstop>leMasterPassNew2</tabstop>
  <tabstop>chkKeepBackup</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsMasterPasswordResetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsMasterPasswordResetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
